src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Cohort/WhenMappingSelectProviderRequestToViewModel.cs

```diff
@@ -1,86 +1,101 @@
-﻿using System.Threading.Tasks;
-using AutoFixture.NUnit3;
+﻿using AutoFixture;
+using Moq;
 using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Api.Client;
+using SFA.DAS.CommitmentsV2.Api.Types.Responses;
 using SFA.DAS.EmployerCommitmentsV2.Web.Mappers.Cohort;
 using SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort;
 using SFA.DAS.Testing.AutoFixture;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace SFA.DAS.EmployerCommitmentsV2.Web.UnitTests.Mappers.Cohort
 {
     [TestFixture]
     public class WhenMappingSelectProviderRequestToViewModel
     {
+        private SelectProviderRequest _request;
+        private Mock<ICommitmentsApiClient> _commitmentsApiClientMock;
+        private AccountLegalEntityResponse _commitmentsApiClientResponse;
+        private SelectProviderViewModelMapper _mapper;
 
-        [Test, AutoData]
-        public async Task ThenMapsReservationId(
-            SelectProviderRequest request,
-            SelectProviderViewModelMapper mapper)
+        [SetUp]
+        public void Setup()
         {
-            var result = await mapper.Map(request);
+            var autoFixture = new Fixture();
+            _request = autoFixture.Freeze<SelectProviderRequest>();
+            _commitmentsApiClientResponse = autoFixture.Freeze<AccountLegalEntityResponse>();
 
-            Assert.AreEqual(request.ReservationId, result.ReservationId);
+            _commitmentsApiClientMock = autoFixture.Freeze<Mock<ICommitmentsApiClient>>();
```

Comment:
just curious why we can't just do 

_commitmentsApiClientMock = new Mock ICommitmentsApiClient ()

All the tests work, but would it work without Freeze? Just curious

Reply:
amended